java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -850,6 +867,17 @@ private static Object[] streamCommands(BaseTransaction<?> transaction) {
             Map.of(
                     streamKey1,
                     Map.of()), // xreadgroup(Map.of(streamKey1, ">"), groupName1, consumer1, options);
+            Map.of(), // xclaim(streamKey1, groupName1, consumer1, 0L, new String[] {"0-1"})
+            Map.of(
+                    "0-3",
+                    new String[] {
+                        "field3", "value3"
+                    }), // xclaim(streamKey1, groupName1, consumer1, 0L, {"0-3"}, options)
+            new String[] {
+                "0-3"
+            }, // xclaimJustId(streamKey1, groupName1, consumer1, 0L, new String[] {"0-3"})
+            new String
+                    [0], // xclaimJustId(streamKey1, groupName1, consumer1, 0L, new String[] {"0-4"}, options)
```

Comment:
spotless, stop this please!
Maybe you can shorten the comment to tweak that?

Reply:
lol